Index.html

```diff
@@ -128,24 +174,31 @@ <h2>About me</h2>
             </ul>
            </div> 
         </div>
-
```

Comment:
- [ ] Kindly edit the alignment of your list of technologies to match the figa template.
### Your design
![2](https://user-images.githubusercontent.com/94551484/206829855-21b8f265-5cab-4da5-b773-db5393d64467.png)
### fIGMA TEMPLATE
![f2](https://user-images.githubusercontent.com/94551484/206829856-fde53ef2-de38-4653-bc75-1b19e166f1bc.png)

Reply:
Thank you. I have added justified the content of the list to center which has aligned them correctly.